test/journeys/specs/oneOnOne/dataApi/guest.js

```diff
@@ -0,0 +1,116 @@
+import {assert} from 'chai';
+
+import testUsers from '@ciscospark/test-helper-test-users';
+
+import setupTestUserJwt from '../../../lib/test-users';
+import {switchToMeet} from '../../../lib/test-helpers/space-widget/main';
+import {
+  sendMessage,
+  verifyMessageReceipt
+} from '../../../lib/test-helpers/space-widget/messaging';
+import {elements as meetElements, declineIncomingCallTest, hangupDuringCallTest} from '../../../lib/test-helpers/space-widget/meet';
+
+if (!(process.env.CISCOSPARK_APPID_ORGID && process.env.CISCOSPARK_APPID_ORGID)) {
+  throw new Error('CISCOSPARK_APPID_ORGID and CISCOSPARK_APPID_ORGID are required to generate guest tokens');
+}
+
+describe('Widget Space: One on One', () => {
+  describe('Data API: Guest', () => {
+    let local, mccoy, remote, spock;
+
+    const mccoyName = 'Bones Mccoy';
+    const spockName = 'Mr Spock';
+
+    before('create test users', () => Promise.all([
+      setupTestUserJwt({displayName: spockName}).then((guestUser) => {
+        spock = guestUser;
+        local = {browser: browser.select('browserLocal'), displayName: spockName, user: guestUser};
```

Comment:
This pattern seems a bit obfuscating-y (that's totally a word). Most of the helper functions only needed the browser and maybe one other piece of information. Instead of referencing this object (e.g. `local.browser`), can we instead use `browserLocal`?

We can move this line down to the 1 test that calls a method that still needs the full object?